build-system/compile/post-closure-babel.js

```diff
@@ -84,9 +84,10 @@ async function postClosureBabel(file) {
   }
 
   debug(CompilationLifecycles['closured-pre-babel'], file);
+  /** @type {?babel.TransformOptions} */
   const babelOptions = babel.loadOptions({caller: {name: 'post-closure'}});
   const {code, map: babelMap} =
-    (await babel.transformFileAsync(file, babelOptions)) || {};
+    (await babel.transformFileAsync(file, babelOptions ?? undefined)) || {};
```

Comment:
`babel.loadOptions` returns `babel.TransformOptions | null` while `babel.transformFileAsync` only accepts `babel.TransformOptions | undefined`

Reply:
Weird mismatch by babel, but this fix works.